tools/osx/crosstool/BUILD.toolchains

```diff
@@ -59,14 +59,18 @@ OSX_TOOLS_CONSTRAINTS = {
         "@platforms//os:tvos",
         "@platforms//cpu:arm64",
     ],
-    "watchos_arm64": [
+    "watchos_sim_arm64": [
         "@platforms//os:watchos",
         "@platforms//cpu:arm64",
     ],
     "watchos_arm64_32": [
         "@platforms//os:watchos",
         "@platforms//cpu:arm64_32",
     ],
+    "watchos_arm64": [
+        "@platforms//os:watchos",
+        "@platforms//cpu:arm64",
+    ],
```

Comment:
This would also have to change in apple_support. Also a breaking change: https://github.com/bazelbuild/apple_support/blob/e81e1ea6d2a089337955c6a956dc3f96ed67ed09/platforms/BUILD#L129-L136

Reply:
Should I also make the change in `platforms/BUILD`?